api/organisations/models.py

```diff
@@ -70,6 +70,15 @@ def over_plan_seats_limit(self):
             self.has_subscription() and 0 < self.subscription.max_seats < self.num_seats
         )
 
+    def over_plan_and_free_seats_limit(self):
+        if self.has_subscription():
+            return (
+                self.has_subscription()
+                and 0 < self.subscription.max_seats < self.num_seats
+            )
+        else:
+            return self.num_seats > 1
```

Comment:
Should maybe make this a constant @matthewelwell ? not sure if that value is defined anywhere?

Reply:
Yep, we can probably just put it on the `Subscription` class or something.

Reply:
Added